plugins/Jobs.cpp

```diff
@@ -76,7 +76,7 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
         //     Atomically dequeues one or more jobs, if available.
         //
         //     Parameters:
-        //     - name - name pattern of jobs to match
+        //     - name - list of name pattern of jobs to match
```

Comment:
I think we need to clarify that if you pass one, you can use `*` but not if you pass many...